api-test/src/main/resources/idRepository/RetrieveIdentityPostV2/RetrieveIdentityPostV2.yml

```
@@ -269,3 +269,7 @@ RetrieveIdentityPostV2:
       }
     ]
 }'
+
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (4 > 0)

(empty-lines)